addons/mail/static/src/core/web/message_patch.js

```diff
@@ -13,27 +13,34 @@ import {
     formatText,
 } from "@web/views/fields/formatters";
 import { useService } from "@web/core/utils/hooks";
+import { usePopover } from "@web/core/popover/popover_hook";
 import { patch } from "@web/core/utils/patch";
+import { AvatarCardPopover } from "@mail/discuss/web/avatar_card/avatar_card_popover";
 
 patch(Message.prototype, {
     setup() {
         super.setup(...arguments);
         this.action = useService("action");
         this.userService = useService("user");
+        this.avatarCard = usePopover(AvatarCardPopover, {
+            closeOnHoverAway: true,
+        });
     },
     getAuthorText() {
-        return this.hasAuthorClickable() ? _t("Open profile") : undefined;
+        return this.hasAuthorClickable() ? _t("Open card") : undefined;
     },
     hasAuthorClickable() {
-        return this.message.author && !this.message.isSelfAuthored;
+        return this.message.author?.user;
```

Comment:
I think we should allow any partner to be clickable, not just users. This would be a regression.